sgemm.cpp

```diff
@@ -1,6 +1,3 @@
-// -*- mode:c++;indent-tabs-mode:nil;c-basic-offset:4;coding:utf-8 -*-
```

Comment:
Why are you removing the mode line? They help auto-configure my text editor.

Reply:
FWIW my vim installation complains about `ft=c++`; it wants `ft=cpp` instead.

Reply:
> Why are you removing the mode line? They help auto-configure my text editor.

Default vim doesn't support filetype "c++", it only supports "cpp" like @cebtenzzre mentioned and vim errors out when I try to open the file. I think you have a custom "c++" filetype on your computer.

Regardless I feel custom text editor configs shouldn't really belong in the repo. It's fine in a company setting where everyone is using the same editor, but for an open source project I don't want to say encourage another contributor to add their Emacs config as well.

Reply:
I don't feel strongly about this. I put the file here so you and other llama.cpp developers can hack on it and make whatever changes you like. So please, by all means. I'm much more interested in your SSSE3 support.